manifests/init.pp

```diff
@@ -180,4 +185,12 @@
     }
   }
 
+  if $enable_grub_hardening {
```

Comment:
On switching back `enable_grub_hardening` -> cleanup changes on grub

Reply:
Done, but I'd slightly worried that this may actually cause problem for people who make these kind of changes with other modules? There's no way to check if the change was done by this module or by something else.